lib/tasks/build.js

```diff
@@ -56,10 +56,22 @@ module.exports.js = function(gulp, config) {
 
 		log.secondary("Browserifying " + src);
 
+		var externals;
+		if (config.externals) {
+			config.bundleExternals = false;
+			externals = config.externals;
+			config.fullPaths = true;
+			delete config.externals;
```

Comment:
because it's not a valid config option for browserify and I don't want to risk leaving it in